p4-16/spec/P4-16-spec.mdk

```diff
@@ -3853,7 +3901,7 @@ struct S {
 
 extern void f<T>(in T data);
 
-f((S){ 10, 20 }); // automatically converted to f((S){a = 10, b = 20});
+f((S){ 10, 20 }); // {10, 20} is converted to {a = 10, b = 20}
```

Comment:
In reality, the tuple is casted to an anonymous struct type but since we don't have an anonymous struct type we cannot say that. So I just stated that the value is converted to a structure-valued expression.